salt/metalk8s/addons/dex/deployed/chart.sls

```diff
@@ -48,19 +75,17 @@ stringData:
     - id: metalk8s-ui
       name: MetalK8s UI
       redirectURIs:
-      - '{% endraw %}https://{{ grains.metalk8s.control_plane_ip }}:8443/oauth2/callback{%
-        raw %}'
+      - "{% endraw %}https://{{ grains.metalk8s.control_plane_ip }}:8443/oauth2/callback{% raw %}"
       secret: ybrMJpVMQxsiZw26MhJzCjA2ut
     - id: grafana-ui
       name: Grafana UI
       redirectURIs:
-      - '{% endraw %}https://{{ grains.metalk8s.control_plane_ip }}:8443/grafana/login/generic_oauth{%
-        raw %}'
+      - "{% endraw %}https://{{ grains.metalk8s.control_plane_ip }}:8443/grafana/login/generic_oauth{% raw %}"
       secret: 4lqK98NcsWG5qBRHJUqYM1
-    enablePasswordDB: true
+    enablePasswordDB: {% endraw -%}{{ dex.spec.localuserstore.enabled }}{%- raw %}
     staticPasswords:
-    - email: admin@metalk8s.invalid
-      hash: $2a$10$2b2cU8CPhOTaGrs1HRQuAueS7JTT5ZHsHSzYiFPm1leZck7Mc8T4W
+    - email: {% endraw -%}{{ dex.spec.localuserstore.email }}{%- raw %}
```

Comment:
Why would we allow changing the 'email' when we don't allow to change the 'username'?

Also, shouldn't this be a *list* of accounts in the config...

Reply:
True, it should be a list of accounts!

I think i left out `username` because the APIserver flag we set was `--oidc-username-claim=email`

Now that I remember that the UI might want to display the username, i will make this field configurable